15:05:20 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 196Object reference not set to an instance of an object.
---------------------------------------
15:09:52 16 December 2014 
   at System.Net.Sockets.Socket.Send(Byte[] buffer)
   at AppedoLT.Core.Trasport.Send(TrasportData objTrasportData) in e:\AppedoLT\AppedoLT.Core\TrasportLayer.cs:line 87
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 162An existing connection was forcibly closed by the remote host
---------------------------------------
15:21:35 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 198Object reference not set to an instance of an object.
---------------------------------------
15:23:41 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 198Object reference not set to an instance of an object.
---------------------------------------
15:24:07 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 133Object reference not set to an instance of an object.
---------------------------------------
15:24:27 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 133Object reference not set to an instance of an object.
---------------------------------------
15:24:37 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 133Object reference not set to an instance of an object.
---------------------------------------
15:24:52 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 133Object reference not set to an instance of an object.
---------------------------------------
15:25:07 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 133Object reference not set to an instance of an object.
---------------------------------------
15:25:22 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 133Object reference not set to an instance of an object.
---------------------------------------
15:25:52 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 197Object reference not set to an instance of an object.
---------------------------------------
15:27:49 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 197Object reference not set to an instance of an object.
---------------------------------------
15:51:07 16 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 197Object reference not set to an instance of an object.
---------------------------------------
18:54:07 18 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
18:57:52 18 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
19:04:39 18 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
19:04:39 18 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 197Object reference not set to an instance of an object.
---------------------------------------
11:50:29 19 December 2014 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 197Object reference not set to an instance of an object.
---------------------------------------
15:45:04 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:00:00 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:21:20 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:31:01 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:36:36 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:43:06 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:51:12 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:52:17 25 December 2014 
   at System.Net.ConnectStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at AppedoLT.BusinessLogic.HttpRequest.ReadResponseBody(String contentType, Int64 contentLength, Stream responseStream, MemoryStream& responseBody) in e:\AppedoLT\AppedoLT.BusinessLogic\HttpRequest.cs:line 615
The request was aborted: The request was canceled.
---------------------------------------
16:52:17 25 December 2014 
   at System.Net.ConnectStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at AppedoLT.BusinessLogic.HttpRequest.ReadResponseBody(String contentType, Int64 contentLength, Stream responseStream, MemoryStream& responseBody) in e:\AppedoLT\AppedoLT.BusinessLogic\HttpRequest.cs:line 615
The request was aborted: The request was canceled.
---------------------------------------
16:52:17 25 December 2014 
   at System.Net.ConnectStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at AppedoLT.BusinessLogic.HttpRequest.ReadResponseBody(String contentType, Int64 contentLength, Stream responseStream, MemoryStream& responseBody) in e:\AppedoLT\AppedoLT.BusinessLogic\HttpRequest.cs:line 615
The request was aborted: The request was canceled.
---------------------------------------
16:53:12 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:54:17 25 December 2014 
   at System.Net.ConnectStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at AppedoLT.BusinessLogic.HttpRequest.ReadResponseBody(String contentType, Int64 contentLength, Stream responseStream, MemoryStream& responseBody) in e:\AppedoLT\AppedoLT.BusinessLogic\HttpRequest.cs:line 615
The request was aborted: The request was canceled.
---------------------------------------
16:54:17 25 December 2014 
   at System.Net.ConnectStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at AppedoLT.BusinessLogic.HttpRequest.ReadResponseBody(String contentType, Int64 contentLength, Stream responseStream, MemoryStream& responseBody) in e:\AppedoLT\AppedoLT.BusinessLogic\HttpRequest.cs:line 615
The request was aborted: The request was canceled.
---------------------------------------
16:54:17 25 December 2014 
   at System.Net.ConnectStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at AppedoLT.BusinessLogic.HttpRequest.ReadResponseBody(String contentType, Int64 contentLength, Stream responseStream, MemoryStream& responseBody) in e:\AppedoLT\AppedoLT.BusinessLogic\HttpRequest.cs:line 615
The request was aborted: The request was canceled.
---------------------------------------
16:55:28 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:56:28 25 December 2014 
   at System.Net.ConnectStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at AppedoLT.BusinessLogic.HttpRequest.ReadResponseBody(String contentType, Int64 contentLength, Stream responseStream, MemoryStream& responseBody) in e:\AppedoLT\AppedoLT.BusinessLogic\HttpRequest.cs:line 615
The request was aborted: The request was canceled.
---------------------------------------
16:58:23 25 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
11:47:42 26 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
12:47:06 30 December 2014 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 259
---------------------------------------
16:46:00 08 January 2015 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 263
---------------------------------------
16:48:37 08 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 198Object reference not set to an instance of an object.
---------------------------------------
16:54:17 08 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 198Object reference not set to an instance of an object.
---------------------------------------
16:54:47 08 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 198Object reference not set to an instance of an object.
---------------------------------------
16:58:47 08 January 2015 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 263
---------------------------------------
17:01:07 08 January 2015 
   at System.Data.SQLite.SQLiteDataReader.VerifyType(Int32 i, DbType typ)
   at System.Data.SQLite.SQLiteDataReader.GetDateTime(Int32 i)
   at AppedoLT.DataAccessLayer.ReportMaster..ctor(String reportName, DateTime ScenarioStartTime, String loagGenName) in e:\AppedoLT\AppedoLT.DataAccessLayer\ReportMaster.cs:line 107Specified cast is not valid.
---------------------------------------
21:39:08 08 January 2015 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 263
---------------------------------------
21:40:33 08 January 2015 
   at System.Data.SQLite.SQLiteDataReader.VerifyType(Int32 i, DbType typ)
   at System.Data.SQLite.SQLiteDataReader.GetDateTime(Int32 i)
   at AppedoLT.DataAccessLayer.ReportMaster..ctor(String reportName, DateTime ScenarioStartTime, String loagGenName) in e:\AppedoLT\AppedoLT.DataAccessLayer\ReportMaster.cs:line 107Specified cast is not valid.
---------------------------------------
22:29:34 08 January 2015 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 263
---------------------------------------
22:30:59 08 January 2015 
   at System.Data.SQLite.SQLiteDataReader.VerifyType(Int32 i, DbType typ)
   at System.Data.SQLite.SQLiteDataReader.GetDateTime(Int32 i)
   at AppedoLT.DataAccessLayer.ReportMaster..ctor(String reportName, DateTime ScenarioStartTime, String loagGenName) in e:\AppedoLT\AppedoLT.DataAccessLayer\ReportMaster.cs:line 107Specified cast is not valid.
---------------------------------------
23:24:54 08 January 2015 
The process cannot access the file 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\out.txt' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at AppedoLT.DataAccessLayer.DataServer.ClearData() in e:\AppedoLT\AppedoLT.DataAccessLayer\DataServer.cs:line 263
---------------------------------------
23:26:19 08 January 2015 
   at System.Data.SQLite.SQLiteDataReader.VerifyType(Int32 i, DbType typ)
   at System.Data.SQLite.SQLiteDataReader.GetDateTime(Int32 i)
   at AppedoLT.DataAccessLayer.ReportMaster..ctor(String reportName, DateTime ScenarioStartTime, String loagGenName) in e:\AppedoLT\AppedoLT.DataAccessLayer\ReportMaster.cs:line 107Specified cast is not valid.
---------------------------------------
23:28:45 08 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 134Object reference not set to an instance of an object.
---------------------------------------
23:32:43 08 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 198Object reference not set to an instance of an object.
---------------------------------------
11:48:36 21 January 2015 
   at System.Data.SQLite.SQLite3.Prepare(SQLiteConnection cnn, String strSql, SQLiteStatement previous, UInt32 timeoutMS, String& strRemain)
   at System.Data.SQLite.SQLiteCommand.BuildNextCommand()
   at System.Data.SQLite.SQLiteCommand.GetStatement(Int32 index)
   at System.Data.SQLite.SQLiteDataReader.NextResult()
   at System.Data.SQLite.SQLiteDataReader..ctor(SQLiteCommand cmd, CommandBehavior behave)
   at System.Data.SQLite.SQLiteCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SQLite.SQLiteCommand.ExecuteNonQuery()
   at AppedoLT.DataAccessLayer.ReportMaster.SetUserRunTime() in e:\AppedoLT\AppedoLT.DataAccessLayer\ReportMaster.cs:line 183SQLite error
index idx_userruntime already exists
---------------------------------------
12:00:02 21 January 2015 
   at System.Net.ConnectStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at AppedoLT.BusinessLogic.HttpRequest.ReadResponseBody(String contentType, Int64 contentLength, Stream responseStream, MemoryStream& responseBody) in e:\AppedoLT\AppedoLT.BusinessLogic\HttpRequest.cs:line 615
The request was aborted: The request was canceled.
---------------------------------------
18:22:44 23 January 2015 
   at AppedoLT.BusinessLogic.VariableManager..ctor(XmlNode variableNode) in e:\AppedoLT\AppedoLT.BusinessLogic\VariableManager.cs:line 66
   at AppedoLTLoadGenerator.RunScenario.Start() in e:\AppedoLT\AppedoLT.LoadGenerator\RunScenario.cs:line 138
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 120Object reference not set to an instance of an object.
---------------------------------------
18:53:03 23 January 2015 
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.Delete(String fullPath, String userPath, Boolean recursive, Boolean checkHost)
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 176Could not find a part of the path 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\966_192_168_1_70'.
---------------------------------------
18:53:13 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
18:55:33 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
18:55:58 23 January 2015 
   at AppedoLT.BusinessLogic.VariableManager..ctor(XmlNode variableNode) in e:\AppedoLT\AppedoLT.BusinessLogic\VariableManager.cs:line 66
   at AppedoLTLoadGenerator.RunScenario.Start() in e:\AppedoLT\AppedoLT.LoadGenerator\RunScenario.cs:line 138
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 120Object reference not set to an instance of an object.
---------------------------------------
18:56:58 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
18:59:35 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
18:59:40 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:00:29 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:01:34 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:02:34 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:03:43 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:05:58 23 January 2015 
   at AppedoLT.BusinessLogic.VariableManager..ctor(XmlNode variableNode) in e:\AppedoLT\AppedoLT.BusinessLogic\VariableManager.cs:line 66
   at AppedoLTLoadGenerator.RunScenario.Start() in e:\AppedoLT\AppedoLT.LoadGenerator\RunScenario.cs:line 138
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 120Object reference not set to an instance of an object.
---------------------------------------
19:08:19 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:09:24 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:14:53 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:15:58 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:16:51 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:17:19 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:17:29 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:17:39 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:18:44 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:18:49 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:18:54 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:19:09 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:20:41 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
19:22:05 23 January 2015 
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.Delete(String fullPath, String userPath, Boolean recursive, Boolean checkHost)
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 176Could not find a part of the path 'E:\AppedoLT\AppedoLT.LoadGenerator\bin\Debug\972_192_168_1_70'.
---------------------------------------
19:22:05 23 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
14:42:22 27 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
14:43:27 27 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
14:44:32 27 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
14:45:32 27 January 2015 
   at AppedoLTLoadGenerator.LoadGenerator.<>c__DisplayClass1.<worker_DoWork>b__0() in e:\AppedoLT\AppedoLT.LoadGenerator\frmLoadGenerator.cs:line 201Object reference not set to an instance of an object.
---------------------------------------
14:48:48 27 January 2015 
   at System.Data.SQLite.SQLite3.Prepare(SQLiteConnection cnn, String strSql, SQLiteStatement previous, UInt32 timeoutMS, String& strRemain)
   at System.Data.SQLite.SQLiteCommand.BuildNextCommand()
   at System.Data.SQLite.SQLiteCommand.GetStatement(Int32 index)
   at System.Data.SQLite.SQLiteDataReader.NextResult()
   at System.Data.SQLite.SQLiteDataReader..ctor(SQLiteCommand cmd, CommandBehavior behave)
   at System.Data.SQLite.SQLiteCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SQLite.SQLiteCommand.ExecuteNonQuery()
   at AppedoLT.DataAccessLayer.ReportMaster.SetUserRunTime() in e:\AppedoLT\AppedoLT.DataAccessLayer\ReportMaster.cs:line 183SQLite error
index idx_userruntime already exists
---------------------------------------
